### DISCUSSION TITLE
Permission to remote denied

### DISCUSSION THREAD

**Discussion by kevkrist** [dendibakh/perf-ninja#93] [Q&A] [answered]
Hi there. I have what feels like a silly question, but after checking out a private branch and committing the changes for the warmup, I am getting the following access permission issue:

```$git push origin private/kkrist
ERROR: Permission to dendibakh/perf-ninja.git denied to kevkrist.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

**Comment by liangzhongkai** [comment]
您的来信我已收到，谢谢

**Comment by dendibakh** [comment, marked as answer]
Please apply here for access:
https://github.com/dendibakh/perf-ninja/blob/main/GetStarted.md#submission-guidelines

**Reply by kevkrist** [reply]
Thank you!
